static/sass/application.scss

```diff
@@ -128,3 +126,58 @@
         transform: rotate(180deg)
     }
 }
+
+/*Datepicker overrides*/
+.ui-widget-header {
+    border: 0 !important;
+    background: #fff !important;
+    color: black !important;
+    font-weight: bold;
+}
+
+.ui-datepicker {
+    margin-top: 20px;
+}
+
+.ui-datepicker table {
+    background: white !important;
+}
+
+.ui-widget-content {
+    border: 1px solid #000000 !important;
+    background: #fff !important;
+}
+
+.ui-state-default {
+    border: 0 !important;
+    background: #fff !important;
+    color: #000 !important;
+    font-weight: normal !important;
+    text-align: center !important;
+}
+
+.ui-state-active {
+    background: #015DA5 !important;
```

Comment:
should be using colours from govuk rather than hard coding values?

Reply:
Good shout.